### PR TITLE
app-misc/cmatrix: update EAPI, DESCRIPTION and HOMEPAGE

### PR DESCRIPTION
**Pull request by gentoo-setan**
Just small updates.

---

Please check all the boxes that apply:

- [x] I can submit this contribution in agreement with the [Copyright Policy](https://www.gentoo.org/glep/glep-0076.html#certificate-of-origin).
- [X] I have certified the above via adding a `Signed-off-by` line to *every* commit in the pull request.
- [X] This contribution has not been created with the assistance of Natural Language Processing artificial intelligence tools, in accordance with the [AI policy](https://wiki.gentoo.org/wiki/Project:Council/AI_policy).
- [X] I have run `pkgcheck scan --commits --net` to check for issues with my commits.

Please note that all boxes must be checked for the pull request to be merged.

### CODE REVIEW

File: app-misc/cmatrix/cmatrix-2.0-r5.ebuild [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+HOMEPAGE="https://github.com/abishekvashok/cmatrix"
+SRC_URI="https://github.com/abishekvashok/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~alpha amd64 ~hppa ppc ppc64 sparc x86"
+IUSE="+unicode"
+
+DEPEND="sys-libs/ncurses:=[unicode(+)?]"
+BDEPEND=""
```
Comment (Flowdalic) [inline app-misc/cmatrix/cmatrix-2.0-r5.ebuild:20]: Drop empty BDEPEND

File: app-misc/cmatrix/cmatrix-2.0-r5.ebuild [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+FONT_SUFFIX="pcf"
+
+inherit cmake font
+
+DESCRIPTION="Ncurses based app to show a scrolling screen from the Matrix"
+HOMEPAGE="https://github.com/abishekvashok/cmatrix"
```
Comment (Flowdalic) [inline app-misc/cmatrix/cmatrix-2.0-r5.ebuild:11]: The homepage URL you dropped,  ttps://www.asty.org/cmatrix/, seems to be still valid. Why drop it?

**Review by Flowdalic — CHANGES_REQUESTED** [review]
Thanks for your contribution.

File: app-misc/cmatrix/cmatrix-2.0-r5.ebuild [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+SRC_URI="https://github.com/abishekvashok/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~alpha amd64 ~hppa ppc ppc64 sparc x86"
```
Comment (Flowdalic) [inline app-misc/cmatrix/cmatrix-2.0-r5.ebuild:16]: Drop keywords to unstable ones

**Review by Flowdalic — CHANGES_REQUESTED** [review]

File: app-misc/cmatrix/cmatrix-2.0-r5.ebuild (hunk 3 above)
Comment (gentoo-setan) [reply app-misc/cmatrix/cmatrix-2.0-r5.ebuild]: done

File: app-misc/cmatrix/cmatrix-2.0-r5.ebuild (hunk 1 above)
Comment (gentoo-setan) [reply app-misc/cmatrix/cmatrix-2.0-r5.ebuild]: done

File: app-misc/cmatrix/cmatrix-2.0-r5.ebuild (hunk 2 above)
Comment (gentoo-setan) [reply app-misc/cmatrix/cmatrix-2.0-r5.ebuild:11]: The website used to be valid but is no longer maintained:

> The current version of CMatrix is 1.2a. 

The current HOMEPAGE is now up to date.

File: app-misc/cmatrix/cmatrix-2.0-r5.ebuild [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+
+FONT_SUFFIX="pcf"
+
+inherit cmake font
+
+DESCRIPTION="Ncurses based app to show a scrolling screen from the Matrix"
+HOMEPAGE="https://github.com/abishekvashok/cmatrix"
+SRC_URI="https://github.com/abishekvashok/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2"
```
Comment (mm1ke) [inline app-misc/cmatrix/cmatrix-2.0-r5.ebuild:14]: this should be `GPL-3+`, see https://github.com/abishekvashok/cmatrix/blob/master/cmatrix.c#L9-L12

Comment (gentoo-setan) [reply app-misc/cmatrix/cmatrix-2.0-r5.ebuild:14]: Good call. Also they got conflicting headers and [readme](https://github.com/abishekvashok/cmatrix?tab=readme-ov-file#page_facing_up-license) license

**Comment by Flowdalic** [comment]
Please squash into a single commit

File: app-misc/cmatrix/cmatrix-2.0-r5.ebuild (hunk 2 above)
Comment (Flowdalic) [reply app-misc/cmatrix/cmatrix-2.0-r5.ebuild:11]: Alright, according to https://www.asty.org/cmatrix/,

> You can find the newest repository for CMatrix at https://github.com/abishekvashok/cmatrix and look for a new official website as well!

it seems sensible to drop  https://www.asty.org/cmatrix/.

**Comment by gentoo-setan** [comment]
I guess this is it @Flowdalic? It's my first squash.

Also should I squash the commits as they come or do it at the end, like now?

**Pull request closed without merging**